docs/products/postgresql/howto/pgbouncer-stats.md

```diff
@@ -2,51 +2,82 @@
 title: Access PgBouncer statistics
 ---
 
-PgBouncer is used at Aiven for [connection pooling](/docs/products/postgresql/concepts/pg-connection-pooling).
+import ConsoleIcon from "@site/src/components/ConsoleIcons"
+
+PgBouncer is used at Aiven as a [connection pooler](/docs/products/postgresql/concepts/pg-connection-pooling) to lower the performance impact of opening new connections to Aiven for PostgreSQL®.
+
+After connecting to PgBouncer, you can check statistics available from PgBouncer, such as:
```

Comment:
What do we mean by "check"? Do I have something to verify or is it just that are displayed?